lib/terraforming/cli.rb

```diff
@@ -5,6 +5,9 @@ class CLI < Thor
     class_option :tfstate, type: :boolean, desc: "Generate tfstate"
     class_option :profile, type: :string, desc: "AWS credentials profile"
     class_option :region, type: :string, desc: "AWS region"
+    class_option :"aws-use-bundled-cert",
```

Comment:
`class_option :use_bundled_cert` looks better.

Underscores are converted to hyphens automatically.

```
  [--use-bundled-cert], [--no-use-bundled-cert]  # Use the bundled CA certificate from AWS SDK
```